Vostok.ServiceDiscovery.Abstractions/Models/ReplicaInfo.cs

```diff
@@ -29,6 +31,8 @@ public ReplicaInfo([NotNull] string environment, [NotNull] string application, [
 
         public string Replica { get; }
 
+        public ITag[] Tags { get; }
```

Comment:
Let's discuss whether it's necessary to extend `IReplicaInfo`. Currently I see no compelling reason to do so :)

Reply:
I don’t understand how to do it right: i didn’t find any other way how to define tags when configuring the beacon without changing `ServiceBeacon` constructors, except that the tags are no longer needed for anything.

Reply:
My basic idea was to implement tag configuration via `IReplicaInfoBuilder` without changing `ServiceBeacon` constructors

Reply:
Confusing... i agree with you that IReplicaInfo must not know about Tags, but for me tags is part of "ServiceBeaconReplicaInfo" model.

Reply:
Look like i really need to configure tags via ServiceBeacon constructor...

Reply:
> My basic idea was to implement tag configuration via IReplicaInfoBuilder without changing ServiceBeacon constructors

Yep, that's fine. Definitely 100x better than messing with beacon constructors. I just wanted to clarify what we'd gain by exposing the result from `IReplicaInfo` directly, why would one access tags from there.

Reply:
Now i see two solutions:
1. Extend `ReplicaInfo` class with Tags property without changing base interface.
2. Add to `IReplicaInfoBuilder` additional method `GetTags` and use it, after ReplicaInfo building, but it looks like a big crutch.

Reply:
okay, i just removed it from `IReplicaInfo` but Tags still defined in `ReplicaInfo`, i can't think of anything better